src/commands/compare.ts

```diff
@@ -28,9 +28,9 @@ export default class Compare extends Command {
     const { flags } = this.parse(Compare);
     const localFlags = await this.mergeFlagsWithDefaults(flags);
     const checker = new BundleChecker(localFlags);
-    const result = await checker.compareByFileExtension();
+    const result = await checker.compareEachFile();
     if (flags.prComment) await checker.commentOnPr(result);
```

Comment:
I think it would be worth separating the checker from git-related features. That could be handled by `utils` (as it already happens)